Scripts/Debug/Yotam LCA Model.py

```diff
@@ -291,21 +283,24 @@ def get_trained_network_multLCA(bipartite_graph, num_features=3, num_hidden=200,
 	mnet.add_projection(projection=pho, sender=hl, receiver=ol)
 
 	# Train network
-	mnet.run(
+	mnet.learn(
 	    inputs=input_set,
 	    minibatch_size=1,
-	    bin_execute=True
+	    bin_execute=True,
+		patience=patience,
+		min_delta=min_delt,
 	)
 
 	for projection in mnet.projections:
-		weights = projection.parameters.matrix.get(mnet)
-		projection.parameters.matrix.set(weights, None)
-
-	mnet.learning_enabled = False
+		try:
+			weights = projection.parameters.matrix.get(mnet)
+			projection.parameters.matrix.set(weights, None)
+		except:
```

Comment:
This should either check 'matrix' in parameters, or narrow down the exception to AttributeError